pkgs/development/python-modules/paramiko/default.nix

```diff
@@ -59,7 +62,7 @@ buildPythonPackage rec {
     mock
     pytestCheckHook
     pytest-relaxed
-  ] ++ lib.flatten (builtins.attrValues passthru.optional-dependencies);
```

Comment:
Why the first change?

Reply:
yeah, I can send that on staging to reduce the rebuild count.

Reply:
On the why:
- pyproject = true is the preferred way over format and all packages should slowly be converted
- optional-dependencies moved out of passthru a while ago
- the functions in lib are preferred over builtins but minor thingy

Reply:
i think his question was only about changing `builtins.attrValues` to `lib.attrValues` as it is just an alias for `builtins`: https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix#L539